Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/MotionEventPresetManager.cpp

```diff
@@ -27,10 +27,11 @@
 
 namespace EMStudio
 {
-    MotionEventPreset::MotionEventPreset(const AZStd::string& name, EMotionFX::EventDataSet&& eventDatas, AZ::Color color)
+    MotionEventPreset::MotionEventPreset(const AZStd::string& name, EMotionFX::EventDataSet&& eventDatas, AZ::Color color, const AZStd::string& comment)
```

Comment:
suggestion: Prefer to pass `AZStd::string` by value here and `AZStd::move` it into the member (same for name). Also consider breaking this line as it's pretty long

Reply:
Just curious why it is better than passing by reference? Is it just different time when the string is allocated, or there is a performance benefit?